core/server/adapters/scheduling/post-scheduling/index.js

```diff
@@ -14,7 +15,7 @@ _private.normalize = function normalize(options) {
 
     return {
         time: moment(object.get('published_at')).valueOf(),
-        url: apiUrl + '/schedules/posts/' + object.get('id') + '?client_id=' + client.get('slug') + '&client_secret=' + client.get('secret'),
+        url: utils.url.urlJoin(apiUrl + '/schedules/posts/' + object.get('id')) + '?client_id=' + client.get('slug') + '&client_secret=' + client.get('secret'),
```

Comment:
This looks a bit confusing.

`utils.url.urlJoin(apiUrl + '/schedules/posts/' + object.get('id'))`

vs.

`utils.url.urlJoin(apiUrl, '/schedules/posts', object.get('id'))`
`utils.url.urlJoin(apiUrl, 'schedules', 'posts', object.get('id'))`